go.mod

```diff
@@ -59,6 +59,7 @@ require (
 	github.com/pelletier/go-toml/v2 v2.2.3
 	github.com/rootless-containers/bypass4netns v0.4.1
 	github.com/rootless-containers/rootlesskit/v2 v2.3.1
+	github.com/shirou/gopsutil/v4 v4.24.7
```

Comment:
This seems to introduce a bunch of other deps that should not be necessary for fixing the issue

Reply:
Thanks, will explore other options

Reply:
> This seems to introduce a bunch of other deps that should not be necessary for fixing the issue

Apologies for the delay, the latest commit implements this feature without the need for other deps. 

I do want to note; it seems like there was a discussion upstream in the containerd cgroups package relating to this issue ([containerd/cgroups#265](https://github.com/containerd/cgroups/pull/265#issuecomment-1418250556)) that never got resolved. Is there alignment for the one of  4 options from that discussion? Should this fix even be implemented in this package or should it be upstream in containerd? 

Let me know your thoughts @AkihiroSuda